packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChartPage.tsx

```diff
@@ -77,6 +82,11 @@ export class HorizontalBarChartPage extends React.Component<IComponentDemoPagePr
               Set the <code>data</code> attribute of IChartDataPoint to specify the benchmark value. The benchmark value
               is shown as an inverted triangle in the chart.
             </p>
+            <h4>AbsoluteScale variant</h4>
```

Comment:
Add some more content about the absolute variant and n/M variant so that it becomes easier for the users to choose 1 vs other. You can take up this change later as well and not block this PR.

Reply:
Ok, I will create a separate PR for that